--- mkstage4.sh.old	2015-05-13 10:45:29.000000000 +0100
+++ mkstage4.sh	2015-05-13 10:48:56.000000000 +0100
@@ -51,7 +51,7 @@
 tarOptions=" --preserve-permissions --create --absolute-names --totals --ignore-failed-read"
  
 # where to put the stage4
-stage4Location=/mnt/backups/stage4
+stage4Location=/mnt/ICYBOX/stage4
  
 # name prefix
 stage4prefix=`hostname`-stage4-`date +\%Y.\%m.\%d`
@@ -68,6 +68,7 @@
 default_exclude_list="
 /dev
 /lost+found
+/media
 /mnt
 /proc
 /sys
@@ -109,7 +110,8 @@
 # include/exclude these files/folders depening on your answer.
 custom_include_list="
 /home/*
-/usr/src/linux-`uname -r`"
+/usr/src/linux-`uname -r`
+/usr/local/portage/packages"
  
 # add files/folders here which are subfolders of a folder listed in $custom_include_list which should NOT
 # be backed up. eg.
